docs(mesh) Landing page and overview (#2289)
Kong/docs.konghq.com: app/mesh/1.0.x/overview.md

## Introduction

<div class="alert alert-ee blue">

>    <b>Demo</b>: To see {{site.mesh_product_name}} in action you can

Reviewer: ```suggestion
   <b>Demo</b>: To see {{site.mesh_product_name}} in action, you can
```

clause requires comma

service behavior to increase application reliability and the efficiency of
your teams.

{{site.mesh_product_name}} is the universal service mesh for enterprise
organizations focused on simplicity and scalability with Kuma and Envoy.
Kong’s service mesh is unique in that it allows you to:

* **Start, secure, and scale with ease**:
  * Deploy a turnkey service mesh with a single command.
  * Group services by attributes to efficiently apply policies.
  * Manage multiple service meshes as tenants of a single control plane to
  provide scale and reduce operational costs.
* **Run anywhere**:
  * Deploy the service mesh across any environment, including multi-cluster,
  multi-cloud, and multi-platform.
  * Manage service meshes natively in Kubernetes using CRDs, or start with a
  service mesh in a VM environment and migrate to Kubernetes at your own pace.
* **Connect services end-to-end**:
  * Integrate into the {{site.ee_product_name}} platform for full stack connectivity,
  including Ingress and Egress traffic for your service mesh.
  * Expose mesh services for internal or external consumption and manage the
  full lifecycle of APIs.

> Thanks to the underlying Kuma runtime, with {{site.mesh_product_name}} you

Reviewer: ```suggestion
Thanks to the underlying Kuma runtime, with {{site.mesh_product_name}}, you
```

can easily support multiple clusters, clouds, and architectures using the
multi-zone capability that ships out of the box. This &mdash; combined with
multi-mesh support &mdash; lets you create a service mesh powered by an Envoy proxy
for the entire organization in just a few steps. You can do this for both
simple and distributed deployments, including multi-cloud, multi-cluster, and
hybrid Kubernetes/VMs:

<center>
  <img src="/assets/images/docs/mesh/multi-zone.jpg" width="600px"/>
  <br>
  <i>{{site.mesh_product_name}} can support multiple zones (like a Kubernetes

>     cluster, VPC, data-center, etc) together in the same distributed deployment.

Reviewer: ```suggestion
    cluster, VPC, datacenter, etc.) together in the same distributed deployment.
```
